MATCHERS.md

```diff
@@ -70,6 +71,36 @@ state: s.enum({values: ['NSW', 'VIC'], name:'state'})
 state: s.enum({values: ['NSW', 'VIC'], name:'state', verbose: true})
 ```
 
+## exactObject
+
+Match an exact object.
+This matcher will not error if any optional properties are left out.
+This matcher will error if any extra properties passed in.
+
+Each property must have a corresponding matcher, and they will be called recursively.
+
+```js
+// match exact object
+person: 'exactObject'
+person: s.exactObject()
```

Comment:
Since this should error if extra properties are found and the above usages aren't looking for any properties, what would this match?

Also what do you think about implementing this as an optional argument to the object matcher instead?